pysteps/visualization/basemaps.py

```diff
@@ -94,35 +115,25 @@ def plot_geography(
     if plot_map == "cartopy" and not CARTOPY_IMPORTED:
         raise MissingOptionalDependency(
             "the cartopy package is required to plot the geographical map "
-            " but it is not installed"
+            "but it is not installed"
         )
 
     if not PYPROJ_IMPORTED:
         raise MissingOptionalDependency(
-            "the pyproj package is required to plot the geographical map"
+            "the pyproj package is required to plot the geographical map "
             "but it is not installed"
         )
 
-    # if plot_map == "cartopy": # not really an option for the moment
-    cartopy_scale = kwargs.get("scale", "50m")
-    cartopy_subplot = kwargs.get("subplot", None)
     crs = utils.proj4_to_cartopy(proj4str)
 
-    # Replace current axis
-    if cartopy_subplot is None:
-        cax = plt.gca()
-        cartopy_subplot = cax.get_subplotspec()
-        cax.clear()
-        cax.set_axis_off()
-
     ax = plot_map_cartopy(
         crs,
         extent,
-        cartopy_scale,
+        scale,

```

Comment:
The `scale` argument is very much specific to Cartopy and a different basemap plotting library might use a different convention, that's why the argument was previously named `cartopy_scale`.